lib/postinstall.js

```diff
@@ -35,7 +35,7 @@ async function getTarget() {
                 'x86_64-apple-darwin';
         case 'win32':
             return arch === 'x64' ? 'x86_64-pc-windows-msvc' :
-                arch === 'arm' ? 'aarch64-pc-windows-msvc' :
+                (arch === 'arm' || arch === 'arm64') ? 'aarch64-pc-windows-msvc' :
```

Comment:
should this just be `arch === 'arm64'` ? I don't think Node.js has `arm` value for windows.

Reply:
I didn't want to cause any regressions but I think you're right.

```suggestion
                arch === 'arm64' ? 'aarch64-pc-windows-msvc' :
```